### PR TITLE
Added weeMoji script and Enhance README.md with WeekMoji Script Details

### PR DESCRIPTION
**Pull request by prynacshrma**
Summary
This pull request updates the `README.md` file to include detailed information about the newly added `weekMoji` script. The changes aim to provide users with an understanding of the script's functionality, usage, and examples.

Changes Made
- Added a section on the `weekMoji` script, including:
  - Overview of the script’s purpose
  - Instructions on how to use the script
  - Examples of the output generated by the script

Benefits
These updates will help users better understand how to utilize the `weekMoji` script effectively, enhancing their experience with the Slackerbot project.

### CODE REVIEW

File: wMoji [hunk 1]
```diff
@@ -0,0 +1,21 @@
+/*
+activation_example:@All
+regex:<@U.*>
```
Comment (wiz0floyd) [inline wMoji:3]: This trigger will be very spammy.

File: emojis_for_weekdays [hunk 2]
```diff
@@ -0,0 +1,22 @@
+/*
+activation_example:@All
+regex:<@U.*>
```
Comment (wiz0floyd) [inline emojis_for_weekdays:3]: This trigger will cause spam.

File: wMoji
Comment (wiz0floyd) [inline wMoji:1]: Duplicate?

File: README.md
Comment (wiz0floyd) [inline README.md:1]: The top level readme file is for the whole bot. Individual parsers do not require a readme. Please review the contributing requirements for this repository.

**Review by wiz0floyd — CHANGES_REQUESTED** [review]

File: wMoji (hunk 1 above)
Comment (prynacshrma) [reply wMoji:3]: changed it

**Comment by wiz0floyd** [comment]
Hi there! I see you marked my comments as resolved, but didn't see any new commits to this branch. Please let me know if you need help with GitHub.

**Pull request closed without merging**